unst_msh_gen/README.md

```diff
@@ -1,32 +1,32 @@
 # Project Name
 Unstructured mesh generation for WW3 using JIGSSAW.
-## Description
+# Description
 Mesh generation script capable of creating unstructured meshes for WW3 global modeling. The tool leverages JIGSAWPY (https://github.com/dengwirda/jigsaw-python) for efficient triangulation.
 Main changes include:
 Implementation of the ocn_ww3.py to create uniform unstructured mesh for global WW3 model.
 This tool is under active development, with future work focused on variable unstructured mesh generation.
 
-## Installation
+# Installation
 
-1- Install jigsawpy (https://github.com/dengwirda/jigsaw-python)
-2- you need following packages:
-	numpy
-        scipy
-	packaging
-	netcdf4
+## 1- Install jigsawpy (https://github.com/dengwirda/jigsaw-python)
 
-3- clone the repo
-        $git clone https://github.com/NOAA-EMC/WW3-tools
-	$cd WW3-tools/unst_msh_gen
+## 2- you need following packages:
```

Comment:
@AliS-Noaa this is different from the list you gave me: 

$ python3 -m pip install scipy
$ python3 -m pip install imageio
$ python3 -m pip install scikit-image
$ python3 -m pip install tifffile
$ python3 -m pip install certifi
$ python3 -m pip install cftime
$ python3 -m pip install netCDF4
$ python3 -m pip install network
$ python3 -m pip install packaging
$ python3 -m pip install pillow
$ python3 -m pip install setuptools

I know some of these are installed already when you install other modules, but wanted to check to see if this list should be updated?